src/nvim/generators/gen_api_dispatch.lua

```diff
@@ -223,6 +223,11 @@ for i = 1, #functions do
           output:write('\n  } else if (args.items['..(j - 1)..'].type == kObjectTypeInteger && args.items['..(j - 1)..'].data.integer >= 0) {')
           output:write('\n    '..converted..' = (handle_T)args.items['..(j - 1)..'].data.integer;')
         end
+        -- accept empty lua tables as empty dictionarys
```

Comment:
"lua" in the above comment is a bit misleading. This applies to all msgpack requests--though it's probably only _useful_ for a Lua client.

Reply:
Right, I was somewhat in "lua-land" when I did this.